hostapd: Radius server support for WPA-PSK

dsinghra123 opened the pull request:
Hostapd provides Radius Server support for WPA-PSK but hostapd.sh lacks
this config option for WPA_PSK. Also, there is no config option for
radius server support for MAC Based authentication. This patch includes
both cases

Signed-off-by: Deependra Rawat deependra.rawat85@gmail.com

dsinghra123 committed:
Merge pull request #1 from openwrt/master

Updating to latest

dsinghra123 committed:
Update hostapd.sh (#2)

dsinghra123 committed:
Update Makefile (#3)

981213 commented:
Hi!
All the issues in your previous attempts (#1559 #1563 ) still exists here.
Please take a look at https://openwrt.org/submitting-patches and https://openwrt.org/docs/guide-developer/working-with-github-pr
squash your commits together and add proper commit description/Sign-off.
GitHub PR description is **not** commit description.

dsinghra123 commented:
There is some issue with my github and I am not able to squash my changes. I will close this pull request and request a new one by addressing all issues mentioned by you.

Thanks.
